dotcom-rendering/src/client/userFeatures/user-features.ts

```diff
@@ -49,18 +55,12 @@ const validateResponse = (
 	response: unknown,
 ): response is UserFeaturesResponse => {
 	if (!isObject(response)) return false;
-
-	if (
-		typeof response.showSupportMessaging === 'boolean' &&
-		isObject(response.contentAccess) &&
-		typeof response.contentAccess.paidMember === 'boolean' &&
-		typeof response.contentAccess.recurringContributor === 'boolean' &&
-		typeof response.contentAccess.digitalPack === 'boolean'
-	) {
-		return true;
-	}
-
-	return false;
+	if (!isBoolean(response.showSupportMessaging)) return false;
+	if (!isObject(response.contentAccess)) return false;
+	if (!isBoolean(response.contentAccess.paidMember)) return false;
+	if (!isBoolean(response.contentAccess.recurringContributor)) return false;
+	if (!isBoolean(response.contentAccess.digitalPack)) return false;
+	return true;
```

Comment:
This might be a matter of taste, but I tend to prefer a single boolean expression for functions like these, and it avoids all of the negations returning false.

Would this suffice?

```suggestion
	return (
		isObject(response) &&
		isBoolean(response.showSupportMessaging) &&
		isObject(response.contentAccess) &&
		isBoolean(response.contentAccess.paidMember) &&
		isBoolean(response.contentAccess.recurringContributor) &&
		isBoolean(response.contentAccess.digitalPack)
	);
```